.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,81 @@
+<component name="InspectionProjectProfileManager">
+  <profile version="1.0">
+    <option name="myName" value="Project Default" />
+    <inspection_tool class="PyPackageRequirementsInspection" enabled="true" level="WARNING" enabled_by_default="true">
```

Comment:
Consider adding additional comments to explain the purpose of the inspection_tool.